cli/command/environment_command.go

```diff
@@ -46,6 +46,10 @@ func (e *EnvironmentCommand) GetCommand() cli.Command {
 						Value: "linux",
 						Usage: "specifies windows or linux container environment cluster",
 					},
+					cli.StringFlag{
+						Name:  "ami",
+						Usage: "specifies a custom AMI to use in the environment",
```

Comment:
Maybe we should specify that the user needs to input an AMI ID, in case the ambiguity inspires somebody to use an AMI name?